invenio_pidstore/models.py

```diff
@@ -69,6 +69,14 @@ class PIDStatus(object):
     should not be reused for something else.
     """
 
+    attributes = [
```

Comment:
Is there perhaps a way to refactor ``PIDStatus`` so we avoid the three repetitions of the attributes? This is not a blocker for me in any way.